packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -130,3 +130,14 @@ export const getChatMessageById = (
   const chatMessagesState = state.pages.chat.messages[chatId]
   return selectById(chatMessagesState, messageId)
 }
+
+export const getPopupMessageId = (state: CommonState) => {
+  return state.pages.chat.popupMessageId
+}
+
+export const isEqualToPopupMessageId = createSelector(
+  [getPopupMessageId, (_state: CommonState, messageId: string) => messageId],
```

Comment:
not sure, but wondering if this in fact does memoize, given the second arg is an unnamed lambda function. I don't know though.
did your test confirm memoization?
curious cuz i havent used this specific pattern before

Reply:
Tbh my first time using reselect. I needed a selector that would accept an arg, docs indicated this was the way to do it.

I confirmed the components are memoized, but not sure how to confirm the selector was memoized. Not strictly necessary to use reselect here but thought it was a good chance to try it out.